paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -420,6 +426,65 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
     }
 
+    fun requiresCvcRecollection(extraRequirements: () -> Boolean): Boolean {
```

Comment:
There's still a lot here, why do we need all of this code in PaymentSheetViewModel?